setup.py

```diff
@@ -44,16 +44,32 @@ def _minimal_ext_cmd(cmd):
             GIT_REVISION = out.decode('utf-8').strip()
         except OSError:
             GIT_REVISION = "Unknown"
+        try:
+            cmd = ['git', 'config', '--get', 'remote.origin.url']
+            out = _minimal_ext_cmd(cmd).decode('utf-8').strip()
+            start_idx = 0
+            for prefix in [b'git@github.com:', b'git://github.com/',
+                           b'http://github.com/', b'https://github.com/',
+                           b'ssh://git@github.com/']:
+                if out.startswith(prefix):
```

Comment:
At this point, `out` has been decoded and is thus a string, and you're using a bunch of `bytes` instances to compare against. I think changing these to not have the `b` in front should fix it. You need to make sure they're either both `bytes` or both `string`, and you should be good..